Util/Xamarin.Build.Download/source/Xamarin.Build.Download.Tests/Test.cs

```diff
@@ -471,7 +487,6 @@ public void TestDisallowUnsafeGetItemsToDownload ()
 					{ "ToFile", "cardview.v7.aar" },
 					{ "RangeStart", "196438127" },
 					{ "RangeEnd", "196460160" },
-					{ "Md5", "b44eb88f7cc621ae616744c6646f5b64" }
```

Comment:
We are removing a check to make sure that the file is both correct/complete as well as the right file. We need to have a new way to do this. Since maven only has SHA1 and MD5, we are probably going to have to do the hashing ourselves on CI or as part of the binding process.